webapp/crashstats/supersearch/views.py

```diff
@@ -284,20 +286,20 @@ def search_custom(request, default_context=None):
         except BadArgumentError as e:
             error = e
 
-    schema = settings.ELASTICSEARCH_INDEX_SCHEMA
-    now = timezone.now()
+    es_crashstorage = build_instance_from_settings(
+        socorro_settings.CRASH_DESTINATIONS["es"]
+    )
 
     possible_indices = []
-    for i in range(26):
-        index = (now - datetime.timedelta(weeks=i)).strftime(schema)
+    for index in es_crashstorage.get_indices():
         possible_indices.append({"id": index, "text": index})
 
     context = default_context
     context["elasticsearch_indices"] = possible_indices
 
     if query:
         context["query"] = json.dumps(query["query"])
-        context["indices"] = ",".join(query["indices"])
+        context["indices"] = ",".join(sorted(query["indices"]))
```

Comment:
I added a `sorted` thing here to stabilize the output so tests passed consistently.